### PR TITLE
Added when: cephx to prevent waiting on keyring with cephx: false

### PR DESCRIPTION
**Pull request by squizzi**
We unnecessarily wait for `/etc/ceph/{{ cluster }}.client.admin.keyring` to be created when `cephx: false` is set in `group_vars/all ` which causes this task to timeout resulting in an abort. 

Adding a `when: cephx` clause fixes this.

### CODE REVIEW

File: group_vars/common-coreoss.sample [hunk 1]
```diff
@@ -14,4 +14,3 @@ dummy:
 #pypy_binary_directory: /opt/bin
 #pip_url: https://bootstrap.pypa.io/get-pip.py
 #local_temp_directory: /tmp
-
```
Comment (leseb) [inline group_vars/common-coreoss.sample]: can you undo this change? this shouldn't be part of your commit. Thanks!

**Comment by leseb** [comment]
Good catch @squizzi please address my comment and we are good to go.

File: group_vars/common-coreoss.sample (hunk 1 above)
Comment (squizzi) [inline group_vars/common-coreoss.sample]: woopsie, no idea how this got in ;) -- I'll fix it

**Comment by leseb** [comment]
test this please
